web/src/main/java/se/inera/intyg/intygsadmin/web/stubs/WCIntegrationRestServiceStub.java

```diff
@@ -86,6 +92,31 @@ public TestCertificateEraseResult eraseTestCertificates(LocalDateTime from, Loca
         return TestCertificateEraseResult.create(0, 0);
     }
 
+    @Override
+    public SendStatusIntegrationResponseDTO sendStatus(SendStatusIntegrationRequestDTO request) {
+        return null;
+    }
+
+    @Override
+    public SendStatusIntegrationResponseDTO sendStatusForCertificates(SendStatusForCertificatesIntegrationRequestDTO request) {
+        return null;
+    }
+
+    @Override
+    public SendStatusIntegrationResponseDTO sendStatusForUnits(SendStatusForUnitsIntegrationRequestDTO request) {
+        return null;
```

Comment:
This null will create a null pointer in the services when you try to take out the count and set it to the response of the controller layer. If we want the stub to work we can return an empty SendStatusIntegrationResponseDTO instead